cfme/configure/access_control.py

```diff
@@ -716,9 +716,13 @@ def delete(self, cancel=True):
                 for currently selected group
         """
         flash_success_msg = 'EVM Group "{}": Delete successful'.format(self.description)
-        flash_blocked_msg = (
-            "EVM Group \"{}\": "
-            "Error during delete: A read only group cannot be deleted.".format(self.description))
+        flash_blocked_msg_list = [
+            ("EVM Group \"{}\": "
```

Comment:
**Required**

Use single quotes so you don't have to escape the double quotes.

```
'EVM Group "{}": "Error during delete: A read only group cannot be deleted.'

Reply:
Good catch. updated